boards/nordic/nrf54h20dk/Kconfig.defconfig

```diff
@@ -7,6 +7,9 @@ choice BT_HCI_BUS_TYPE
 	default BT_HCI_IPC if BT
 endchoice
 
+config NRFS
+	default y if USB_DEVICE_STACK_NEXT
```

Comment:
Why not 
```
	select NRFS if NRFS_HAS_VBUS_DETECTOR_SERVICE
	select NRFS_VBUS_DETECTOR_SERVICE_ENABLED if NRFS_HAS_VBUS_DETECTOR_SERVICE
```
under `config UDC_DWC2` in `drivers/usb/udc/Kconfig.dwc2` instead?

The selects have the advantage that `NRFS` cannot be disabled if `UDC_DWC2` is enabled. With just this default, the `NRFS` can be disabled leading to build time failure.

Reply:
Changed as suggested.